File: motor3d 2.0.ConvertedToC#/ObjetoGeometrico3D.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.Linq;
using System.Xml.Linq;
namespace Motor3D.Espacio3D
{
	public abstract class ObjetoGeometrico3D : ObjetoGeometrico
	{

		public new event ModificadoEventHandler Modificado;
		public new delegate void ModificadoEventHandler(ref ObjetoGeometrico3D Sender);

		public ObjetoGeometrico3D() : base()
		{
		}

		public override string ToString()
		{
			return base.ToString();
		}
	}
}

File: motor3d 2.0/objetogeometrico3d.vb
Namespace Espacio3D
    Public MustInherit Class ObjetoGeometrico3D
        Inherits ObjetoGeometrico

        Public Shadows Event Modificado(ByRef Sender As ObjetoGeometrico3D)

        Public Sub New()
            MyBase.New()
        End Sub

        Public Overrides Function ToString() As String
            Return MyBase.ToString()
        End Function
    End Class
End Namespace